config/model_configs/sphere_aquaplanet_rhoe_equilmoist_allsky_gw_raw_zonallyasymmetric.yml

```
@@ -9,7 +9,7 @@ dt: "400secs"
 surface_setup: "DefaultMoninObukhov"
 t_end: "15hours"
 non_orographic_gravity_wave: true
-vert_diff: true
+vert_diff: "DecayWithHeightDiffusion"
```

```suggestion
vert_diff: "FriersonDiffusion"
```
(We should keep one ci test with FriersonDiffusion to maintain the code. It doesn't matter which one it is.)